newIDE/app/src/MainFrame/Editors/StartPage/index.js

```diff
@@ -36,6 +40,9 @@ const styles = {
   },
 };
 
+//TODO
+//Forcé l'ouverture de onCreate, mais avec l'onglet "example"
```

Comment:
This could be done using two classic solution in React:
1. "lifting" the state of the dialog showing the starter/examples into the MainFrame, storing a `createDialogCurrentTab` variable in MainFrame `state` so that the MainFrame own the current tab and is able to change it. This is usually what we do in React. It's called a **controlled** component. You pass this `createDialogCurrentTab` state as a prop to the create dialog, and you pass a callback ("changeCurrentTab") that is called when the user click on a new tab. Finally, you remove the state variable containing the tab inside the create dialog (and you replace it by a prop).

This pattern of removing the state from somewhere to put it in the parent is called **lifting the state**. 
But this would complicate MainFrame and I think that it's better to keep MainFrame unaware of these tabs as possible. So instead, we can do another solution:

2. Store an additional `state` variable in MainFrame which is "createDialogInitialTab" (notice that the name is now "initial" instead of "current"). Then, pass just this prop to the create dialog (no need for a callback). In the constructor/initialization of the create dialog, you use this props to know what to put in the state of the current tab.

When you want to open the dialog, you call `openCreateDialog` in MainFrame, passing an additional argument which is the "initialTab", that you put in the state "createDialogInitialTab". This will result in the dialog opening (because you also set `createDialogOpen` to true in openCreateDialog) AND the initial tab being used by `CreateProjectDialog` to know what the initial tab is.

```
state = {
    currentTab: this.props.initialTab,
  };
```

This is called an **uncontrolled component** (because you can pass it a few details, but then the component is managing its state itself, without any "help" from its parent).

Reply:
It's "classic" react stuff here, so not a priority for this PR but can help you to understand the logic behind React :)